tests/container_disk_test.go

```diff
@@ -87,7 +87,8 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 		vmi.Spec.Volumes[0].ContainerDisk.ImagePullPolicy = policy
 		vmi = tests.RunVMIAndExpectScheduling(vmi, 60)
 		Expect(vmi.Spec.Volumes[0].ContainerDisk.ImagePullPolicy).To(Equal(expectedPolicy))
-		pod := libvmi.GetPodByVirtualMachineInstance(vmi, vmi.Namespace)
+		pod, err := libvmi.GetPodByVirtualMachineInstance(vmi, vmi.Namespace)
```

Comment:
I know that @EdDev will not agree with me but I think as long as we don't have usage for the return error we should rather assert/panic inside the helper function. Otherwise, we need to do what is done in this commit -> pollute testing code(imho without advantage).

Reply:
I don't insist. I approve as long as I can get https://github.com/kubevirt/kubevirt/pull/8902#discussion_r1047371583